apm-agent-core/src/main/java/co/elastic/apm/agent/logging/ApmServerLogAppender.java

```diff
@@ -75,7 +76,7 @@ public static ApmServerLogAppender createAppender(@PluginAttribute("name") Strin
             INSTANCE = new ApmServerLogAppender(name, layout);
         }
 
-        return new ApmServerLogAppender(name, layout);
+        return INSTANCE;
```

Comment:
[for reviewer] this was the dumb mistake that made it fail, creating a singleton and returning another value.